Configuration/Generator/python/QCD_Pt_1800_2400_13TeV_TuneCUETP8M1_cfi.py

```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
```

Comment:
please update to TuneCP5
`from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *`